install-and-configure/advanced-configuration/gpu.md

```diff
@@ -0,0 +1,55 @@
+# NVIDIA GPU Monitoring Configurations
+
+## Monitoring GPU utilization
+
+In order for Kubecost to understand a container's GPU utilization percentage, Kubecost depends on metrics being available from NVIDIA [DCGM Exporter](https://github.com/NVIDIA/dcgm-exporter). Follow the below instructions to install and configure DCGM Exporter on each of your GPU-enabled clusters.
+
+```sh
+helm upgrade -i dcgm dcgm-exporter \
+  --repo https://nvidia.github.io/dcgm-exporter/helm-charts
+  -n dcgm-exporter --create-namespace \
+  --set serviceMonitor.enabled=false
+```
+
+For more advanced configurations, optionally follow the steps below:
+
+```sh
+# 1. Upgrade the NVIDIA Device plugin to its latest release. Additionally enable GPU feature discovery.
+helm upgrade -i nvdp nvidia-device-plugin \
+  --repo https://nvidia.github.io/k8s-device-plugin \
+  -n nvidia-device-plugin --create-namespace \
+  --set gfd.enabled=true
```

Comment:
This one needs more explanation. Users may have to first uninstall (manually) their device plugin when using cloud providers, who do not install it using Helm. While it's good to be on the latest plugin, I'm not confident yet of making this a carte blanche recommendation for all users on all platforms as I don't know what the side effects might be. Perhaps a message saying something to the effect of, "here's how to deploy the device plugin with GFD enabled (which implicitly deploys NFD), you may also choose to modify your existing device plugin deployment, if applicable, to install GFD. Otherwise, check with your cloud provider."